packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownContentProvider.tsx

```diff
@@ -0,0 +1,44 @@
+import { createContext, useCallback, useState } from 'react';
+
+export type DropdownContentContextValue<Key extends string = string> = {
+  currentContentId: string | null;
+  onContentChange: (key: Key) => void;
+  resetContent: () => void;
+};
+
+export const DropdownContentContext =
+  createContext<DropdownContentContextValue>({
+    currentContentId: null,
+    onContentChange: () => {},
+    resetContent: () => {},
+  });
+
+type DropdownContentProviderProps = {
+  children: React.ReactNode;
+};
+
+export const DropdownContentProvider = ({
```

Comment:
I'm not sure about this API, do we want the dropdown to handle it's content? can't we pass it as a <children>? Does not feel right

Reply:
general comment about these changes on dropdown UI behavior